submissions/Lu-sure/Popup/index.html

```diff
@@ -0,0 +1,168 @@
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/apps.png" alt="Apps icon" class="popup-item-img">
+                      Apps
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/documents.png" alt="Documents icon" class="popup-item-img">
+                      Documents
+                    </a>
+                  </li>
+                </ul>
+
+                <input type="checkbox" id="more-button" class="hide-checkbox more" >
+                <label for="more-button" class="more-bt-label" ><span class="more-bt-text">More</span></label>
+
+                <div class="popup-divider"></div>
+
+                <ul class="popup-list add-popup">
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/chat.png" alt="Chat icon" class="popup-item-img">
+                      Chat
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/classroom.png" alt="Classroom icon" class="popup-item-img">
+                      Classroom
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/collection.png" alt="Collection icon" class="popup-item-img">
+                      Collection
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/mars.png" alt="Mars icon" class="popup-item-img">
+                      Mars
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/blogs.png" alt="Blogs icon" class="popup-item-img">
+                      Blogs
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/adds.png" alt="Blogs icon" class="popup-item-img">
+                      Ads
+                    </a>
+                  </li>
+
+                  <li class="popup-bottom">
+                    <a href="#" class="a">Even more from Kottans</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </li>
+
+          <li>
+            <a href="#" class="a nav-img-item bell-item">
+              <img class="bell-img" src="navigation-icons/bell.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" class="a nav-img-item">
+              <img class="menu-img round-img" src="navigation-icons/user.jpg" alt="">
```

Comment:
Why ´alt´ attribute of navigation icons is empty, but inside popup are filled.

Reply:
I did so because the `alt` text for navigation icons makes total mess on menu bar, so I added descriptive image name/path. If it is not justified, I will add alternative text to that icons.

Reply:
It depends on your aim. If you want to promote your icon images in search fill `alt` attribute if not leave it empty. Also, don't forget about screen reader users)

Reply:
I will leave it empty for this time. For screen readers I added descriptive path.

Reply:
Ok, it's up to you)